www/classes/Model/User.php

```diff
@@ -155,6 +155,39 @@ public function update(string $name, int $perm_level, bool $active, int $alt_css
         return User::getById($this->id);
     }
 
+    public function combineUsers(User $user1): ?User
```

Comment:
Is handig denk ik om hier even een beschrijving boven te zetten. Zodat we makkelijk kunnen teruglezen wat er precies wordt overgezet van welke user ID naar welke.